Makefile

```diff
@@ -56,4 +56,4 @@ dev:
 	$(DEPS_BIN_DIR)/mocha test/*-test.js $(MOCHA_FLAGS) --watch
 
 watch:
-	$(DEPS_BIN_DIR)/babel -w src --out-dir lib
+	$(DEPS_BIN_DIR)/babel -w src --out-dir lib -s inline
```

Comment:
I definitely see the value in source maps here, but I think we should only have it in the `watch` target, which is used during development.